### PR TITLE
A few changes related to work on picker and arms

### PR DESCRIPTION
**Pull request by mmcadamsms**

### CODE REVIEW

File: smokey/src/main/java/frc/robot/commands/ArmPlusPickerUptakeCommand.java [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+
+    private static final double cubeInitialSpeed = 0.75;
+    private static final double cubeSteadyStateSpeed = 1.0;
+    private static final double cubeOverCurrentInitialLimitAmps = 60.0;
+    private static final double cubeOverCurrentSteadyStateLimitAmps = 35.5;
+
+    private final ArmSubsystem armSubsystem;
+    private final EveryBotPickerSubsystem pickerSubsystem;
+    private final PowerDistributionPanelWatcherSubsystem watcherSubsystem;
+    private boolean isTargetCone = true;
```
Comment (josieammer) [inline smokey/src/main/java/frc/robot/commands/ArmPlusPickerUptakeCommand.java:41]: would this be better handled by an Enum for game piece type?

File: smokey/src/main/java/frc/robot/control/AutonomousChooser.java [hunk 2]
```diff
@@ -128,8 +134,8 @@ public Command getScoreRoutine(Pose2d NodePosition){
         if(this.subsystems.getPickerSubsystem() != null) {
             command.addCommands(new ManipulatePickerCommand(subsystems.getPickerSubsystem(), true));
         }
-        else if (this.subsystems.getEveryBotPickerSubsystem() != null) {
-            command.addCommands(new EveryBotPickerAutoExpellCommand(subsystems.getEveryBotPickerSubsystem()));
+        else if (this.subsystems.getEveryBotPickerSubsystem() != null) {// cube uses uptake command to expell
+            command.addCommands(new EveryBotPickerAutoUptakeCommand(subsystems.getEveryBotPickerSubsystem()));
```
Comment (josieammer) [inline smokey/src/main/java/frc/robot/control/AutonomousChooser.java:138]: Maybe this command needs a new name.  Maybe UptakeConeExpelCube?
Or we could make one command that is EveryBotPickerCommand that takes an Enum ExpelCube, ExpelCone, UptakeCone, UptakeCube?

**Review by josieammer — APPROVED** [review]
There are a few things I would like to clean up (see comments), but OK to merge this and tackle those in a later PR.

File: smokey/src/main/java/frc/robot/commands/ArmPlusPickerUptakeCommand.java (hunk 1 above)
Comment (mmcadamsms) [reply smokey/src/main/java/frc/robot/commands/ArmPlusPickerUptakeCommand.java:41]: Likely yes ... good idea

File: smokey/src/main/java/frc/robot/commands/ArmPlusPickerUptakeCommand.java [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+        if(this.isTargetCone) {
+            this.fillListForCone();
+        }
+        else {
+            this.fillListForCube();
+        }
+    }
+
+    @Override
+    public void execute() {
```
[the remark this answers is not in the document]
Comment (mmcadamsms) [reply smokey/src/main/java/frc/robot/commands/ArmPlusPickerUptakeCommand.java:86]: A few reasons:
1. I wanted to be able to disable the default watcher on the motor power port for the picker motor
2. I wanted to have more control over the current maximum for the picker motor - especially at motor startup
3. I wanted to be able to stop the motion when overcurrent happens - detect the presence of the game piece and be done with the command

File: smokey/src/main/java/frc/robot/control/AutonomousChooser.java (hunk 2 above)
Comment (mmcadamsms) [reply smokey/src/main/java/frc/robot/control/AutonomousChooser.java:138]: I think we should decide the UX here so we can accommodate how we think we can best get muscle memory for co-pilot

**Pull request merged**